web/src/admin/providers/ProviderWizard.ts

```diff
@@ -58,6 +62,7 @@ export class ProviderWizard extends AKElement {
                 }}
             >
                 <ak-wizard-page-type-create
+                    name="selectProviderType"
```

Comment:
Makes it a lot easier for the tests to find!  And hey, this is actually a control: ak-radio-card.  We should abstract that.